geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/vector/SimpleFeatureVector.scala

```diff
@@ -245,7 +245,7 @@ object SimpleFeatureVector {
       val isLong = dateVector.exists(_.isInstanceOf[BigIntVector])
       if (isLong) { Encoding.Max } else { Encoding.Min }
     }
-    val encoding = SimpleFeatureEncoding(fidEncoding, geomPrecision, datePrecision)
+    val encoding = SimpleFeatureEncoding(fidEncoding, geomPrecision, datePrecision, flipAxisOrder = false)
```

Comment:
- Any way to derive the axis order from the `StructVector`?

Reply:
arrow vectors can have metadata, so we could put something in there - but otherwise, probably not.

Reply:
That's fine, for the purpose of encoding this is not necessary at this time.

Reply:
That's fine, for the purpose of encoding this is not required can keep as is.